app/scripts/models/notifications.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * The notifier broadcasts messages across multiple channels (iframe, tabs, browsers, etc).
+ */
+
+'use strict';
+
+define([
+  'backbone',
+  'underscore'
+], function (Backbone, _) {
+
+  var EVENTS = {
+    PROFILE_IMAGE_CHANGE: 'profile:image:change'
+  };
+
+  var Notifications = Backbone.Model.extend({
```

Comment:
@zaach - Does this need to be a model, or is this really just a channel Composite with some extra goodness? Channels in lib/channels include Backbone.Events, I'm wondering if this module is conceptually similar enough to be moved over there.